packages/organization-viewer-cli/src/index.ts

```diff
@@ -1,65 +1,114 @@
+      'thegraph',
+      { daoSubgraphUrl: 'https://api.thegraph.com/subgraphs/name/0xgabi/dao-subgraph-rinkeby' }
+    ],
+    signer: {},
+  })
+
+  console.log('\nOrganization:')
   const org = connection.organization(ORG_ADDRESS)
+  console.log(org.toString())
+
+  return org
+}
 
+async function inspectOrg(org: Organization): Promise<void> {
   console.log('\nPermissions:')
   const permissions = await org.permissions()
   permissions.map((permission: Permission) => console.log(permission.toString()))
 
+  console.log('\nA role from a permission:')
+  const role = await permissions[4].getRole()
+  console.log(role.toString())
+
   console.log('\nApps:')
   const apps = await org.apps()
   apps.map((app: App) => console.log(app.toString()))
 
   console.log('\nA voting app:')
-  app = apps.find((app: App) => app.name == 'voting')!
-  console.log(app.toString())
+  const votingApp = apps.find((app: App) => app.name == 'voting')!
+  console.log(votingApp.toString())
 
   console.log('\nA repo from an app:')
-  const repo = await app.repo()
+  const repo = await apps[2].repo()
   console.log(repo.toString())
 
   console.log('\nAn app by address:')
-  app = await org.app(apps[1].address)
-  console.log(app.toString())
+  const appByAddress = await org.app(apps[1].address)
+  console.log(appByAddress.toString())
 
   console.log('\nAn app from a permission:')
-  app = await permissions[1].getApp()
-  if (app) { console.log(app.toString()) }
+  const appFromPermission = await permissions[1].getApp()
+  if (appFromPermission) { console.log(appFromPermission.toString()) }
+}
 
-  console.log('\nA role from a permission:')
-  const role = await permissions[1].getRole()
-  console.log(role.toString())
+async function inspectVotingHighLevel(org: Organization): Promise<void> {
+  const apps = await org.apps()
+  const app = apps.find((app: App) => app.name == 'voting')!
+
+  console.log('\nVoting:')
+  const voting = new Voting(app, 'https://api.thegraph.com/subgraphs/name/ajsantander/voting-subgraph')
+  console.log(voting.toString())
+
+  console.log('\nVotes:')
+  const votes = await voting.votes()
+  votes.map((vote: Vote) => console.log(vote.toString()))
+
+  console.log('\nCasts:')
+  const vote = votes[0]
+  const casts = await vote.casts()
+  casts.map((cast: Cast) => console.log(cast.toString()))
+
+  console.log('\nAnalysis of a vote:')
+  console.log(`Vote for "${vote.metadata}" was ${vote.executed ? "executed" : "not executed"}, with ${vote.yea} yeas and ${vote.nay} nays.`)
+  const voters = casts.map((cast: Cast) => cast.voter)
+  console.log('Voters:', voters)
 }
 
-function initConnection(): Connection {
-  // return aragonConnect({
-  //   connector: ['json', { data }],
-  //   signer: {},
-  // })
+async function inspectVotingLowLevel(org: Organization): Promise<void> {
+  console.log('\nLow-level inspection of a voting app:')
+  const wrapper = new GraphQLWrapper('https://api.thegraph.com/subgraphs/name/ajsantander/voting-subgraph')
```

Comment:
Is this only for the demo, or do you see a reason why users would want to use `GraphQLWrapper` versus their own GraphQL library?

Reply:
This is something Gabi requested, the ability to interact directly with the subgraph. I would prefer to use this because it has pretty good error tracking features. Beyond that, its just a graphql wrapper, as the name implies.